frontend/src/components/layout/Body/Body.jsx

```diff
@@ -39,25 +49,35 @@ function Body({
   hideFooter,
   dashboard,
   menu,
+  userRole,
 }) {
+  const user = useUser()
+  const isValidRole = checkRole(user, userRole)
+
   return (
-    <StyledBody className="styledBody">
-      {hideHeader && dashboard ? (
-        ''
+    <div>
```

Comment:
Usar React.Fragment en lugar de <div> => `<></>`

Reply:
Con Fragment me salía un error (Fragments should contain more than one child...) así que no lo he envuelto con nada, aunque con <div> sí que funcionaría.